.github/workflows/pin-requirements.yml

```
@@ -34,6 +30,20 @@ jobs:
         with:
           ref: main
 
+      - name: Get latest tag
+        id: latest_tag
+        run: echo "::set-output name=tag::$(git describe --tags --abbrev=0)"
+
+      - name: Set SIGSTORE_RELEASE_TAG and SIGSTORE_NEW_BRANCH
+        run: |
+          if [ -n "${{ inputs.tag }}" ]; then
+            echo "SIGSTORE_RELEASE_TAG=${{ inputs.tag }}" >> $GITHUB_ENV
+            echo "SIGSTORE_NEW_BRANCH=pin-requirements/sigstore/${{ inputs.tag }}" >> $GITHUB_ENV
+          else
+            echo "SIGSTORE_RELEASE_TAG=${{ steps.latest_tag.outputs.tag }}" >> $GITHUB_ENV
+            echo "SIGSTORE_NEW_BRANCH=pin-requirements/sigstore/${{ steps.latest_tag.outputs.tag }}" >> $GITHUB_ENV
+          fi
```

```suggestion
            echo "SIGSTORE_RELEASE_TAG=${INPUT_TAG}" >> $GITHUB_ENV
            echo "SIGSTORE_NEW_BRANCH=pin-requirements/sigstore/${INPUT_TAG}" >> $GITHUB_ENV
          else
            echo "SIGSTORE_RELEASE_TAG=${LATEST_TAG}" >> $GITHUB_ENV
            echo "SIGSTORE_NEW_BRANCH=pin-requirements/sigstore/${LATEST_TAG}" >> $GITHUB_ENV
          fi
```
@woodruffw like this?